NickvisionMoney.GNOME/Blueprints/account_settings_dialog.blp

```diff
@@ -81,6 +81,54 @@ Adw.MessageDialog _root {
             activates-default: true;
           }
         }
+
+        Adw.ActionRow _customDecimalSeparatorRow {
+          title: _("CustomCurrencyDecimalSeparator.Field");
+
+          [suffix]
+          Gtk.DropDown _customDecimalSeparatorDropDown {
+            valign: center;
+            show-arrow: false;
```

Comment:
Why'd you hide the arrows for these properties? I get the design choice, but I feel like for an accessibility stand point it might make it easier for the user to know it is a drop down with the arrow there. Unless there is some GNOME HIG I don't know about that says hidden the arrow is ok :)

Reply:
The only reason is that I wanted to save space for when "Other" is selected and there are row label, dropdown and entry in one line in pretty narrow dialog. I will think how to change it to look better and keep arrows. Probably I will change entries placeholder to something short so it will not require much space, it's not possible to enter more than 2 characters so it's kinda waste currently.